blog-posts/takeuntil-takewhile/takeWhile-takeUntil-takeWhat.md

```diff
@@ -0,0 +1,106 @@
+---
+published: false
+title: "TakeWhile, TakeUntil, TakeWhat?"
+cover_image: ""
+description: "The RxJS TakeWhile and TakeUntil operator already sound similar and behave similar. So what's the difference?"
+tags: rxjs, javascript, typescript, webdev
+series:
+canonical_url:
+---
+
+It's already well known, that there are situations where you actually want to unsubscribe from your Observables.
```

Comment:
I would rephrase this. You don't "want" to unsubscribe. One MUST unsubscribe unless 100% sure it's not needed.

My personal opinion is that storing and using the subscription is the only safe way, that you can easily explain to new RX users.  There are no "but" or "if" that surround that.

Reply:
I would try to avoid implications like "It's already well known". Instead maybe use something like "It might be know to you already that..."